x/stake/keeper/delegation.go

```diff
@@ -688,7 +693,10 @@ func (k Keeper) CompleteUnbonding(ctx sdk.Context, delAddr sdk.AccAddress, valAd
 	if ubd.CrossStake {
 		events, err = k.crossDistributeUndelegated(ctx, delAddr, valAddr)
 		if err != nil {
-			return ubd, sdk.Events{}, err
+			if !(sdk.IsUpgrade(sdk.BCFusionSecondHardFork) &&
```

Comment:
Can we define an error for "not enough funds to cover relay fee" ?

Reply:
Is this will lead to a hardfork?

Reply:
it's only happened after the second hardfork

Reply:
error defined